### PR TITLE
Added integration test for BIDS downloads

### PR DESCRIPTION
**Pull request by constellates**
@gsfr @rentzso Here's the integration test to cover the BIDS upload and download functionality. It tests that the file structure is preserved for both full file downloads and symlinked downloads.

Let me know if you have any questions, concerns or requests.

Thanks

### CODE REVIEW

**Comment by gsfr** [comment]
Thanks, @constellates. We're separately setting up automated integration test infra in #146.

@rentzso: Please make sure this PR is set up to work with our new infra.

File: test/integration_tests/test_bids_download.py [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+    session.params = {
+        'user': 'test@user.com',
+        'root': True
+    }
+
+    # Convert file list to a tree
+    dataset = file_list_to_tree(file_list)
+
+    # Create a group
+    test_data.group_id = 'test_group_' + str(int(time.time()))
```
Comment (rentzso) [inline test/integration_tests/test_bids_download.py]: please change this line to
`test_data.group_id = 'test_group_' + str(int(time.time()*1000))`
we then avoid to use the same group for two different test.

**Comment by rentzso** [comment]
@constellates the test is compliant with our infra.
It is only needed to change how the group_id is generated

**Comment by constellates** [comment]
@rentzso Thanks for taking a look. I've added the group ID update.

**Comment by rentzso** [comment]
this has landed on https://github.com/poldracklab/bids-core

Closing

**Pull request closed without merging**